saleor/core/analytics.py

```diff
@@ -15,7 +18,8 @@
 
 def get_client_id(request):
     parts = [request.META.get(key, '') for key in FINGERPRINT_PARTS]
-    return uuid.uuid5(UUID_NAMESPACE, '_'.join(parts))
+    encoded_parts = '_'.join(parts).encode('utf_8')
+    return uuid.uuid5(UUID_NAMESPACE, str(encoded_parts))
```

Comment:
This looks like it would behave differently in Python 2 and 3. An explanatory comment would be nice.